docs/manual/download.md

```diff
@@ -23,7 +23,7 @@ The C++ library is distributed as two separate packages for Windows Desktop and
 - [Windows Desktop package `Microsoft.MixedReality.WebRTC.Native.Desktop`](https://www.nuget.org/packages/Microsoft.MixedReality.WebRTC.Native.Desktop)
 - [Windows UWP package `Microsoft.MixedReality.WebRTC.Native.UWP`](https://www.nuget.org/packages/Microsoft.MixedReality.WebRTC.Native.UWP)
 
-The C++ packages contain the shared library `Microsoft.MixedReality.WebRTC.Native.dll` as well as its import library (`.lib`).
+The C++ NuGet packages contain the shared library `Microsoft.MixedReality.WebRTC.Native.dll` as well as its import library (`.lib`).

```

Comment:
Missing rename here?

Reply:
Ooh, is the thing that all NuGet pkgs still use the old name?

I'd rather write this kind of comments from the point of view of 2.0, using `mrwebrtc.lib`, and add a note block like "In versions before 2.0, mrwebrtc.dll is named Microsoft.MixedReality.WebRTC.Native.dll. The following steps are necessary...". So a) we don't need to come back here and change this when we generate new NuGet packages, b) it's cleared what's the norm and what's outdated, and c) we can remove those blocks more easily if they become obsolete.

Reply:
No, there's currently no NuGet package containing an `mrwebrtc.dll` file.

Reply:
Ok, makes sense. But indeed currently NuGet packages all use the old name.

Reply:
I actually expanded and clarified the differences between `v1.0` and `v2.0` for the C++ library. How does that look?

Reply:
Much easier to follow now.